### PR TITLE
Implement the ability to show projection data with error margin

### PR DESCRIPTION
**Pull request by benib**
- This adds the ability to show projection data with error margin
- It adds a new boolean to the schema `isProjection`
- depending on the value of `isProjection` different inputs in the single party elements are shown
- either one number for the exact result or 3 numbers for lower bound, bestGuess, upper bound
- the value of `isProjection` also drives the decision which layout to render

- fixture data, test and doc is part of this PR

- deployed on test env

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,13 @@ There is only one endpoint to get `renderingInfo` in this tool:
 
 This endpoint returns a renderingInfo object with static HTML/CSS/SVG that is fully responsive.
 
+### Projection data
+
+In case `isProjection` is true, the rendering looks different to show the error margin.
+![Election projection data](https://github.com/nzzdev/Q-election-votes/blob/master/readme-images/votes_projection.png)
+
+If `isProjection` is set to true, the parties `percentage` input is hidden, instead thre inputs `projection.lower`, `projection.bestGuess` and `projection.upper` are shown. These should be used to enter the projection best guess as long as the lower and upper bound of the error margin.
```
Comment (manuelroth) [inline README.md]:
```suggestion
If `isProjection` is set to true, the parties `percentage` input is hidden, instead the inputs `projection.lower`, `projection.bestGuess` and `projection.upper` are shown. These should be used to enter the projection best guess as long as the lower and upper bound of the error margin.
```

File: views/HtmlStatic.html [hunk 2]
```diff
@@ -187,13 +240,22 @@
         return enhancedParties.others;
       },
       threshold: ({ item }) => {
+        debugger;
+        console.log('threshold', item.threshold);
```
Comment (manuelroth) [inline views/HtmlStatic.html]: This should probably be removed

File: resources/locales/en/translation.json [hunk 3]
```diff
@@ -12,6 +12,8 @@
   "Parteifarbe": "party color",
   "Farbcode": "color code",
   "CSS Klassenname": "CSS class name",
+  "min. Stimmen in %": "min. votes in %",
+  "max. Stimmen in %": "max. votes in %",
   "Stimmen in %": "votes in %",
```
Comment (manuelroth) [inline resources/locales/en/translation.json]: There are some missing translations for other strings like "Hochrechnungsdaten" and "Stimmen in %"

**Comment by benib** [comment]
@manuelroth thanks for the hints. these are now corrected.

**Pull request merged**